avocado/utils/asset.py

```diff
@@ -132,7 +132,11 @@ def _download(self, url_obj, asset_path):
                 self._create_hash_file(asset_path)
                 return self._verify_hash(asset_path)
         finally:
-            os.remove(temp)
+            try:
+                os.remove(temp)
+            except FileNotFoundError:
+                LOG.info("Temporary asset file unavailable due to failed"
```

Comment:
maybe replace LOG.info to LOG.warn ? Not sure.

Reply:
As deleting the temporary file should not impact the test because the download failed anyway and a message about it will be displayed, I though just inform is fine. I would go with a `pass`, but I noticed it is not a good practice.